sdk/highlight-java/highlight-common/src/main/java/io/highlight/sdk/common/HighlightOptions.java

```diff
@@ -88,6 +90,19 @@ public Builder version(String version) {
 			return this;
 		}
 
+
+		/**
+		 * Sets the service name for the highlight options being constructed. <br>
+		 * If not set, the default value is <b>"unknown"</b>.
+		 *
+		 * @param serviceName the service name to set
+		 * @return this builder
+		 */
+		public Builder serviceName(String serviceName) {
+			this.serviceName = serviceName;
+			return this;
+		}
+
```

Comment:
can we document the new behavior in https://github.com/highlight/highlight/blob/36fa7cbcb5f0017c100ea2525575b36b44913a03/docs-content/sdk/java.md#L4?